src/config.example.ts

```diff
@@ -2,6 +2,7 @@ export default {
   serverUrl: 'ws://localhost:8080',
   mapUrl: (mapName: string) =>
     `https://amrl.cs.utexas.edu/amrl_maps/${mapName}/${mapName}.vectormap.json`,
+  mapDirUrl: `https://amrl.cs.utexas.edu/amrl_maps/maps_directory.txt`,
```

Comment:
based on ut-amrl/amrl_maps#3, you probably want to change this to `.json`

Reply:
:male_detective:  This was un-testable due to CORS stuff, good catch